src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -266,6 +275,8 @@ public void ifPullBeforeOverridePresent(Consumer<PullBeforeOverrideOption> consu
       errors.add("Missing date range start definition");
     if (!isDateRangeValid())
       errors.add(INVALID_DATE_RANGE_MESSAGE);
+    if (!isPemFilePresent())
```

Comment:
This wouldn't be right as is: An `ExportConfiguration` is valid without a PEM file. Pem file is only required for encrypted forms. Maybe we can tell the `ExportConfiguration` if it's a configuration for an encrypted form or not and use that to compute its validness

Reply:
Ok got you.